package.json

```diff
@@ -39,6 +39,7 @@
     "@types/mocha": "^7.0.2",
     "@types/node": "^14.0.14",
     "@types/proxyquire": "^1.3.28",
+    "@types/rewire": "^2.5.28",
```

Comment:
I tend to pin dependencies and configure renovate for bumps. No yarn lock needs to be pushed etc.

Reply:
Our team does not have a guide about handling deps (i think). @helio-frota is anything we can do about it ??

Reply:
@alexalikiotis I think we leave as it for now :+1: we can revisit when nodejs-reference-architecture is finished or when the team add some entry about this :+1: